internal/workload/wrapper.go

```diff
@@ -131,7 +141,7 @@ func (ww *Workload) RegisterObserver(observer Observer) {
 
 func (ww *Workload) Init() error {
 	// Enable auto-update podman timer:
-	svc, err := service.NewSystemdRootless("podman-auto-update", nil, false)
+	svc, err := service.NewSystemd("podman-auto-update", nil, service.SystemBus, nil)
```

Comment:
question: why podman-auto-update need SystemBus and not UserBus?
(I'm aware this part is just a refactor, but still trying to understand this part since we handle the other podman's service as UserBus)

Reply:
No idea and both are valid services. I did not want to change it because I was unsure and I didn't want to make more changes than needed. But in essence I agree that it looks to be more for the `userBus` than the `systemBus`, maybe @machacekondra can chime in and share his thoughts as well.